classes/Cart.php

```diff
@@ -1092,9 +1092,11 @@ public static function cacheSomeAttributesLists($ipa_list, $id_lang)
             ORDER BY ag.`position` ASC, a.`position` ASC'
         );
 
+        $colon = Context::getContext()->getTranslator()->trans(': ', [], 'Shop.Theme.Catalog);
```

Comment:
```suggestion
        $colon = Context::getContext()->getTranslator()->trans(': ', [], 'Shop.Pdf');
```
Again, I am sorry but after a double check with @LouiseBonnard she realized wordings on invoices are exceptions and should be located in Shop.Pdf.